docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -134,35 +214,13 @@ you can force the broker to reload images from your org.
 apb bootstrap
 ----
 
-== Using the mobile-service tag
-
-For MCP enabled _APBs_, make sure we have a `mobile-service` tag inside the `apb.yml` file:
-
-....
-...
-tags: 
-  - mobile-service
-....
-
-== Using the servicename label
-
-Inside of the `apb.yml` file, make sure you use the `serviceName:` label, like:
-
-....
-...
-metadata:
-  displayName: Aerogear Sync Server
-  console.openshift.io/iconClass: icon-nodejs
-  serviceName: fh-sync-server
-...
-....
-
 == Testing an APB
 
-Currently, there are not many tools for testing. The APB team has a few open Github issues and proposals:
+Any roles used in your APB should have a test playbook, provision, deprovision and bind.
```

Comment:
Maybe _Good practice is to have a special playbook for testing, called `test.yml`. This playbook is used for quick verification of the implemented roles within the APB repository. If you include some/all roles (provision, bind, deprovision, unbind) into this playbook, you can easily verify their functionality simply by running `apb test` command._

Reply:
:+1: changed